components/userRequest.tsx

```diff
@@ -51,43 +58,49 @@ const UserRequestDashboardItem: React.FunctionComponent<UserRequest> = ({
       throw new Error(err.message);
     }
   };
+
+  // only show users emailVerified that are null
   return (
     <div>
-      <div className="grid grid-cols-4 gap-10 justify-items-start m-5">
-        <div className="flex flex-row">
-          <div className="w-10 h-10 mr-4 bg-placeholder rounded-full" />
-          <div className="w-32">
-            <p className="font-display self-center">{name}</p>
-            <p>User Role</p>
-          </div>
-        </div>
-        <div>
-          <p className="self-center font-normal">{email}</p>
-        </div>
-        <div>
-          <p className="self-center font-normal">{phoneNumber}</p>
-        </div>
-        <div>
-          <div className="flex space-x-4">
+      <div className="grid grid-cols-4 gap-2 justify-items-start m-5">
+        <Link href={`invite/userAccountPage/${id}`}>
+          <div className="flex space-x-40 bg-white hover:bg-placeholder">
+            <div className="flex flex-row">
+              <div className="w-10 h-10 mr-4 bg-placeholder rounded-full" />
+              <div className="w-32">
+                <p className="font-display self-center">{name}</p>
+                <p>User Role</p>
```

Comment:
replace this with the actual user role - maybe reference how Shannon did it in ./componments/userDashboard